src/wizard/Wizard.container.tsx

```diff
@@ -58,6 +53,15 @@ function WizardContainer({
     revision: interfaceRevision,
   });
 
+  // TODO: Remove before merge - it's just for testing to make it easier for reviewer :-)
+  // if (data) {
+  //   const params = data!.interface!.revision!.spec!.input.parameters[0];
+  //   data!.interface!.revision!.spec!.input.parameters = [params, {
+  //     name: "foo",
+  //     typeRef: params.typeRef,
+  //   }]
+  // }
```

Comment:
To delete before merge. @mszostok  You can use this snippet to test multiple input params, as we don't have a real case (yet) 🙂